WeWatch/WeWatch/ContentView.swift

```diff
@@ -10,12 +10,11 @@ import SwiftUI
 struct ContentView: View {
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundStyle(.tint)
+            
             Text("Hello, world!")
+                .font(Font.poppinsBold30px)
+            Image("home-active-icon")
```

Comment:
Apply the same approach used for fonts and colors to images/icons, enabling their use in code like: `Image(.homeActiveIcon)`